src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,133 @@
+package duke;
+import exceptions.DukeDeadlineException;
+import exceptions.DukeEventException;
+import exceptions.DukeException;
+
+import java.io.IOException;
+import java.time.LocalDate;
+
+/**
+ * Parser for user input command
+ */
+public class Parser {
+
+
+    /**
+     * Parse all given String input to determine respective corresponding commands that the user is calling
```

Comment:
```suggestion
     * Parses all given String input to determine respective corresponding commands that the user is calling
```

In method header comments, the first sentence should start in the form Parses..., Sends ..., Adds ... (not Parse or Parsing etc.)